airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/writer/BaseS3Writer.java

```diff
@@ -34,7 +34,7 @@
  * <li>Log and close the write.</li>
  * </ul>
  */
-public abstract class BaseS3Writer implements S3Writer {
+public abstract class BaseS3Writer implements DestinationFileWriter {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(BaseS3Writer.class);
   public static final String DEFAULT_SUFFIX = "_0";
```

Comment:
I guess we could change it to private as constant is used only inside this class :)